extending/template_system/info_graph.rst

```diff
@@ -0,0 +1,178 @@
+.. _template_info_graph:
+
+
+Graph output for :command:`conan info` command
+==============================================
+
+.. warning::
+
+    This has to be an considered as an **experimental** feature, we might
+    change the context provided to this templates once we have more exmpales
+    from the community.
+
+
+The :ref:`conan_info` command can generate a visualization of the dependency graph, it
+comes in two flavors: `html` and `dot` (GraphViz), but both take the same template
+parameters. Conan will use the following input files, if found, inside the Conan
+cache folder:
+
+ * HTML output: *<cache>/templates/output/info_graph.html*.
+ * DOT output: *<cache>/templates/output/info_graph.dot*.
+
+Context
+-------
+
+These files should be valid Jinja2 templates and they will be feed with the following
+context:
+
+ * ``base_template_path``: path to the directory where the choosen template file is
```

Comment:
is it required to be absolute path, or can be relative (to what?)

Reply:
This variable is provided by Conan. It is the absolute path to the template file.

When the user overrides the default template (which is in-memory, not a file) this variable `base_template_path` will be equal to `<cache>/templates/output` (`<cache>` is the actual path to the cache folder) so the user can use it to provide a path to assets: something like this in the template `{{ base_template_path }}/css/myfile.css` which will be rendered as the absolute path `<cache>/templates/output/css/myfile.css` and it will work regardless of the folder where the generated file is located.

For the template provided by Conan by default this will be equal to `None`, the default template is not a file, it is in-memory.